.gitignore

```diff
@@ -11,4 +11,5 @@ docs/.yarn/
 !.yarn/versions
 .next
 .idea
-.vscode
\ No newline at end of file
+.vscode
```

Comment:
Kind of against adding your OS and IDE files to the ignore of everyone using the project with other things.